source/GameInterface/Services/MobileParties/Messages/AddNewTroop.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Common.Messaging;
+
+namespace GameInterface.Services.MobileParties.Messages
+{
+    /// <summary>
+    /// Event to tell GameInterface to handle unit being recruited
+    /// </summary>
+    public record AddNewTroop : IEvent
```

Comment:
This is a command
![image](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/assets/15619189/78aa76a5-52b2-41ae-8624-2d3daeb6d7ff)

```suggestion
    public record AddNewTroop : ICommand
```